pkg/discovery/service.go

```diff
@@ -128,14 +128,17 @@ func (discovery *Discovery) Refresh(ctx context.Context) error {
 			if err != nil {
 				discovery.log.Error("error updating node uptime in statdb")
 			}
+			err = discovery.cache.Delete(ctx, node.Id)
+			if err != nil {
+				discovery.log.Error("error deleting unresponsive node from cache:", zap.Error(err))
```

Comment:
no need to duplicate the "error" message everywhere, it's already in the log level